Python/optimizer_analysis.c

```diff
@@ -276,6 +276,20 @@ sym_is_null(_Py_UOpsSymType *sym)
     return (sym->flags & (IS_NULL | NOT_NULL)) == IS_NULL;
 }
 
+static inline bool
```

Comment:
Could you delete `is_const` and `get_const` (way below) please? (And replace all their usages in the DSL with these.)